lib/archethic/utils/regression/benchmarks/helpers/tps_helper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule ArchEthic.Utils.Regression.Benchmarks.Helpers.TPSHelper do
+  @moduledoc """
+  Helpers Methods to carry out the TPS benchmarks
+  """
+
+  def withdraw_uco_from_testnet_endpoint(recipient_address) do
```

Comment:
Any particular reason you want to benchmark the testnet endpoint because you can always use the localhost endpoint to get a better picture since any requests to testnet endpoint suffer from network hop latencies.